runtime/src/launch/gasnetrun_common/gasnetrun_common.h

```diff
@@ -39,12 +39,15 @@ static char** chpl_launch_create_argv(const char *launch_cmd,
   const int largc = 5;
   char *largv[largc];
 
+  snprintf(_nlbuf, sizeof(_nlbuf), "%d", numLocales);
+
   largv[0] = (char *) launch_cmd;
   largv[1] = (char *) "-n";
-  snprintf(_nlbuf, sizeof(_nlbuf), "%d", numLocales);
   largv[2] = _nlbuf;
-  largv[3] = (char*) "-E";
-  largv[4] = chpl_get_enviro_keys(',');
+  largv[3] = (char *) "-N";
+  largv[4] = _nlbuf;
+  largv[5] = (char*) "-E";
+  largv[6] = chpl_get_enviro_keys(',');
```

Comment:
`const int largc`, above, sizes `largv[]` and now needs to be at least 7.

Reply:
Shoot rebasing error -- good catch